platform-core-module/src/main/java/com/softicar/platform/core/module/page/PageContentDiv.java

```diff
@@ -20,13 +20,13 @@ public class PageContentDiv extends DomDiv implements IEmfPageContentElement, ID
 
 	private final Optional<IEmfPage<?>> page;
 
-	public PageContentDiv(PageNavigationLink<?> link) {
+	public PageContentDiv(PageNavigationEntry<?> linkEntry) {
```

Comment:
```suggestion
	public PageContentDiv(PageNavigationEntry<?> navigationEntry) {
```

Reply:
Not really. The current naming tries to imply that the entry designates a "link", as opposed to a "folder". That terminology is used all over the place.

Reply:
well then, still not perfect, but anyways